app/population/middleware/set-results.population-table.js

```diff
@@ -0,0 +1,30 @@
+const presenters = require('../../../common/presenters')
+const locationsFreeSpacesService = require('../../../common/services/locations-free-spaces')
+
+async function setResultsPopulationTable(req, res, next) {
+  try {
+    const { dateRange, filter } = req
+
+    const freeSpaces = await locationsFreeSpacesService.getPrisonFreeSpaces({
+      dateFrom: dateRange[0],
+      dateTo: dateRange[1],
+      filter,
+    })
+
+    const query = req.query
```

Comment:
This could be destructured together on line 6

Reply:
Good catch! Fixed :)